api/src/pcapi/routes/public/individual_offers/v1/events.py

```diff
@@ -72,6 +73,9 @@ def post_event_offer(body: events_serializers.EventOfferCreation) -> events_seri
     Create Event Offer
     """
     venue_provider = authorization.get_venue_provider_or_raise_404(body.location.venue_id)
+    venue = offerers_repository.find_venue_by_id(body.location.venue_id)
+    if not venue:
+        raise api_errors.ApiErrors({"location.venueId": f"Venue with id {body.location.venue_id} does not exist."})
```

Comment:
Détail, le message standard c'est plutôt `"Resource cannot be found"`, et même si ce n'est pas complètement homogène dans le projet, on met plutôt le message dans un array car potentiellement il peut il y en avoir plusieurs 
```suggestion
        raise api_errors.ApiErrors({"location.venueId": ["Venue cannot be found"])
```